src/provisioner/provisioner.py

```diff
@@ -56,11 +60,13 @@ def config_apply(solution_conf_url: str, cortx_conf_url: str = None):
 
     @staticmethod
     def config_apply_cortx(cortx_config_store):
+        Log.info('Applying cortx config')
```

Comment:
Messages need to be meaningful. Which cortx config is applied is not answered.

Reply:
Done.